build/scripts-2.7/hollow

```diff
@@ -0,0 +1,100 @@
+#!/nfs/homes4/chenou/anaconda3/envs/Born/bin/python
+# Hollow --- http://hollow.sourceforge.net
+# Copyright (c) 2009 Bosco Ho & Franz Gruswitz
+
+from hollow import __version__
+from hollow.core import make_hollow_spheres, defaults, help
+
+def load_optional_psyco_module():
+  try:
+    import psyco
+    psyco.full()
+    print "Loaded optional PSYCO module for speed"
+  except:
+    pass
+  
+def is_running_in_idle():
+  return not '__file__' in globals()
+
+if __name__ == "__main__":
+  import sys
+  import optparse
+
+  copyright = """
+  Hollow %s (c) 2009 Bosco Ho & Franz Gruswitz.
+
+  Generates a PDB of fake atoms that fill voids, pockets,
+  clefts and channels of a protein structure.
+  """ % __version__
+
+  parser = optparse.OptionParser(usage="usage: %prog [options] pdb")
+  
+  parser.add_option(
+      "-g", dest="grid_spacing", type="float", 
+      help="grid spacing in angstroms " \
+        + "(default %.2f angstroms;" % defaults.grid_spacing \
+        + " suggested 0.2 for final resolution)",
+      default=defaults.grid_spacing)
+  parser.add_option(
+      "-c", dest="constraint_file",
+      help="read CONSTRAINT_FILE for grid constraints", default='')
+  parser.add_option(
+      "-o", dest="out_pdb", help="output hollow spheres to OUT_PDB" ,
+      default='')
+  parser.add_option(
+      "-p", dest="interior_probe", type="float",
+      help="radius of ball to explore cavities"
+      " (default %.2f angstroms = 95%% x radius"
+      " of output atom type suggested)" \
+        % defaults.interior_probe,
+      default=defaults.interior_probe)
+  parser.add_option(
+      "-s", dest="surface_probe", type="float",
+      help="radius of probe to roll over surface" \
+        + " used to define depressions" \
+        + " (default %.2f angstroms)" % defaults.surface_probe,
+      default=defaults.surface_probe)
+  parser.add_option(
+      "-w", dest="process_waters",
+      action="store_true", 
+      help="process water molecules" \
+        + " (no-flag=remove waters from surface calculation;" \
+        + " flag=include waters in protein surface)",
+      default=not defaults.is_skip_waters)
+  parser.add_option(
+      "-b", dest="bfactor_probe", type="float",
+      help="radius around a grid point, in which the" \
+        + " b-factors of heavy atoms are averaged (0.0=off;" \
+        + " suggested=4.0; default=%.f)" % defaults.bfactor_probe,
+      default=defaults.bfactor_probe)
+  parser.add_option(
+      "-i", dest="is_hollow",  
+      help="Enable/disable the grid.exclude_surrounded " \
+        + "default is yes"  \
+        + "switch to no if one wants interior filled grid",
+      default='yes')
```

Comment:
Make it a boolean switch (choose the proper type with `action="store_false"` ) and set the default as `True`.